File: Facebook.NET - VB/Contracts/GraphAPI.PricePoint.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class PricePoint
            <DataMember(Name:="user_price")>
            Public Property UserPrice() As String
            <DataMember(Name:="user_price")>
            Public Property Credits() As System.Nullable(Of Double)
            <DataMember(Name:="user_price")>
            Public Property LocalCurrency() As String
        End Class
    End Class
End Namespace

File: Facebook.NET/Contracts/GraphAPI.PricePoint.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class PricePoint
        {
            [DataMember(Name = "user_price")]
            public string UserPrice { get; set; }

            [DataMember(Name = "user_price")]
            public double? Credits { get; set; }

            [DataMember(Name = "user_price")]
            public string LocalCurrency { get; set; }
        }
    }
}